controllers/pvc.go

```diff
@@ -154,11 +160,12 @@ func (r *VolumeSnapshotBackupReconciler) buildPVCClone(pvcClone *corev1.Persiste
 		if pvcClone.Spec.StorageClassName == nil {
 			pvcClone.Spec.StorageClassName = sourcePVC.Spec.StorageClassName
 		}
-
 		pvcClone.Spec.Resources = sourcePVC.Spec.Resources
 
 		// use the clonedPVCSize that is computed earlier
-		pvcClone.Spec.Resources.Requests.Storage().Set(int64(clonedPVCSize))
+		r.Log.Info(fmt.Sprintf("buildPVCClone: use the clonedPVCSize that is computed earlier"))
+		pvcClone.Spec.Resources.Requests.Storage().Set(clonedPVCSize.Value())
+		r.Log.Info(fmt.Sprintf("buildPVCClone: updated size for pvcClone: %v", pvcClone.Spec.Resources.Requests.Storage().Value()))
```

Comment:
I think updated size log here is redundant with https://github.com/migtools/volume-snapshot-mover/pull/251/commits/e6d01c533fadd83f3502f107984c0a6cb3daf02d#diff-f62131094710ef63a76f5e484071a4e7254ff8a2d176f0957e2dd677495eff04R123 from earlier.

I think there is three logs to indicate pv size was updated..

Reply:
perhaps turn some of them to debug logs is my 2cents.